### DISCUSSION TITLE
Portfolio ToolKit

### DISCUSSION THREAD

**Discussion by mandar-gite** [OpenBB-finance/OpenBB#4527] [Q&A]
What is the template in which the portfolio accepts the data?

**Comment by northern-64bit** [comment]
Hello @mandar-gite!
Here's an example of a valid portfolio: https://www.dropbox.com/s/03wjjf1lfkqjmtn/holdings_example.xlsx?dl=0. This can be modified to ones use. Make sure to move it to `portfolio/holdings` directory within the [OpenBBUserData](https://docs.openbb.co/terminal/guides/advanced/data) directory, before trying to load it through the terminal. 

If you use the SDK you can load the portfolio with the following code:
```python
from openbb_terminal.sdk import openbb
import pandas as pd

P = openbb.portfolio.load(
  transactions_file_path = '/Users/path_to/OpenBBUserData/portfolio/holdings/holdings_example.xlsx',
  benchmark_symbol = 'VTI',
  full_shares = False,
  risk_free_rate = 3.0
)
```

You can also create a new one by your own. Here's some info about the requirements from the official documentation page: https://docs.openbb.co/terminal/guides/intros/portfolio#how-to-use
> Note that the Excel sheet requires the following columns:
> * **Date** - The date the trade occurred
> * **Name** - The name of the security
> * **Type** - The type of the security. Use Stock/Crypto/ETF as appropriate
> * **Price** - The price the security was added or removed at, on a per-unit basis
> * **Quantity** - How much of the security in question was added or removed
> * **Side** - Whether you bought or sold. Use Buy/Deposit/1 to add to the portfolio or Sell/Withdrawal/0 to remove from the portfolio a search criteria, country, sector or industry.
> 
> The template Excel file also has additional columns but these are optional. The OpenBB Terminal can figure out by itself what industry, sector, country and region belongs to the loaded in Equity. So the field can be left blank if your holdings do not include this information.

Here's the guide to using the portfolio menu for the terminal referenced above: https://docs.openbb.co/terminal/guides/intros/portfolio; and here's the SDK portfolio menu introduction: https://docs.openbb.co/sdk/guides/intros/portfolio

Feel free to ask if you want a more detailed explanation, clarifications or just have other questions.
